core/app.py

```diff
@@ -555,9 +555,13 @@ def invoke_custom_command(self):
                 # a workaround to make the damn thing work.
                 exepath, args = match.groups()
                 path, exename = op.split(exepath)
-                subprocess.Popen(exename + args, shell=True, cwd=path)
+                p = subprocess.Popen(exename + args, shell=True, cwd=path, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+                output = p.stdout.read()
+                logging.info("Custom command %s %s:\n %s", exename, args, output)
```

Comment:
All other log entries are single line, I would prefer to keep these entries single line as well.

Reply:
Updated, thanks